testng-1.1.dtd.php

```diff
@@ -0,0 +1,59 @@
+<?
```

Comment:
No needed. Github page doesn't serve php.

It should be replaced by some js code

Reply:
So shall I get rid of this file and also the other php file as well ?

Reply:
Yes, you can.